internal/resolver/testdata/test-project2/mta.yaml

```diff
@@ -0,0 +1,502 @@
+_schema-version: 2.1.0
```

Comment:
This looks exactly the same like in test-project folder. Instead of copying the file maybe use the same file with a different env file (that contains different values) and just pass it a different expected result?

Reply:
I removed the mta.yaml from project2 and used the original one in my new test